styling-and-themes/theme-swatches.md

```diff
@@ -14,76 +14,92 @@ Telerik UI for Blazor comes with three [built-in (base) themes]({%slug general-i
 
 #### In this article:
    * [Basics](#basics)
-   * [Buit-in themes and swatches list](#buit-in-themes-and-swatches-list)
+   * [Built-in themes and swatches list](#built-in-themes-and-swatches-list)
 
 
 ## Basics
 
 The theme swatches are different color variations of the base themes. While they use the same variables as the built-in theme they accompany, they have different color values to deliver a variety of shades for the Telerik components. Any of the base themes and their swatches can be easily customized as desired - read more about that in the [Custom Themes article]({%slug themes-custom%}).
 
-You can find the built-in themes and their swatches in the following resources:
+You can find the built-in themes and all their swatches in the following resources:
```

Comment:
This sentence here may mislead the developer that they need to get familiar with the ThemeBuilder or kendo-themes in order to use swatches. Maybe we should clarify that these resources are for optional custom themes. If we want to help customers preview the looks of the swatches, better link the online demos. At this part of the page, the reader needs to find out the "default" use case scenario - how the swatches look and how to obtain them.

Reply:
My concern is that they cannot explore all the available swatches in the demos, only 1/3 of them. Plus, we initially wanted to list where all the swatches can be found in general - https://github.com/telerik/blazor/issues/3041#issue-999244769.